tests/views/test_survey.py

```diff
@@ -10,6 +10,8 @@
 from response_operations_ui import app
 from response_operations_ui.controllers.survey_controllers import get_survey_short_name_by_id
 
+survey_ref = '222'
```

Comment:
This variable is only used once, we should either just put it directly into where it's being used or replace the other instances of `222` with `survey_ref`